DevEdu.API/DevEdu.Business.Tests/Data/GroupData.cs

```diff
@@ -5,5 +5,6 @@ public class GroupData
         public const int ExpectedAffectedRows = 1;
         public const int GroupId = 1;
         public const int MaterialId = 1;
+        public const int LessonId = 1;
```

Comment:
it's a bad practice to have such values as consts
it's better to declare them in each test method separately

Reply:
1